libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1122,25 +1125,25 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 					"))\n";
 				IRVariable selectorVariable(m_context.newYulVariable(), *TypeProvider::fixedBytes(4));
 				define(selectorVariable, hashVariable);
-				m_code << "mstore(" << to_string(CompilerUtils::freeMemoryPointer) << ", " << freeMemoryPre << ")\n";
+				m_code << m_utils.allocateUnboundedFunction() << "(" << freeMemoryPre << ", 0)\n";
```

Comment:
Hmm? `allocateUnbounded` does not seem to take two arguments?

Reply:
A test coverage gap!

Reply:
Did you add a test case for this?

Reply:
Test is here: https://github.com/ethereum/solidity/pull/10776